packages/ibm-products/src/components/NonLinearReading/NonLinearReading.mdx

```diff
@@ -0,0 +1,76 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
```

Comment:
These docs should move to `NonLinearReading.docs-page.js` and you can delete this file.